build/configs/imxrt1020-evk/SAM/defconfig

```diff
@@ -1135,7 +1135,6 @@ CONFIG_SMARTFS_MAXNAMLEN=32
 CONFIG_SMARTFS_ALIGNED_ACCESS=y
 # CONFIG_SMARTFS_DYNAMIC_HEADER is not set
 CONFIG_SMARTFS_JOURNALING=y
-# CONFIG_SMARTFS_JOURNAL_VERIFY is not set
 # CONFIG_SMARTFS_SECTOR_RECOVERY is not set
```

Comment:
this change is redundant

Reply:
We are doing away with CONFIG_SMARTFS_JOURNAL_VERIFY.
This will not be required as a separate config with the new journaling model.